test/unit/auth/auth-api-request.spec.ts

```diff
@@ -534,6 +564,75 @@ describe('FirebaseAuthRequestHandler', () => {
     });
   });
 
+  describe('getAccountInfoByPhoneNumber', () => {
+    const httpMethod = 'POST';
+    const host = 'www.googleapis.com';
+    const port = 443;
+    const path = '/identitytoolkit/v3/relyingparty/getAccountInfo';
+    const timeout = 10000;
+    it('should be fulfilled given a valid phoneNumber', () => {
+      const expectedResult = {
+        users : [
+          {phoneNumber: '+11234567890'},
```

Comment:
Should we also add the `providerUserInfo` field to the expected response, and see if get parsed correctly?